test_site/home.htm

```diff
@@ -1,6 +1,18 @@
 <html>
   <head>
     <title>Home Page</title>
+    <script type="text/javascript">
```

Comment:
To avoid bloating the homepage more could we have this as a standalone page where more "shadow-rooty" stuff can happen in the future (I imagine this will only grow)